### PR TITLE
feat(refs dplan-4020): Use permission

### PR DESCRIPTION
**Pull request by salisdemos**
use newly added Permission in TWIG to hide the documents area where needed
and adjust Layout

### CODE REVIEW

**Review by sierrazamudiodemos — APPROVED** [review]
I tested it works, thanks

File: client/js/components/map/admin/AdminLayerList.vue [hunk 1]
```diff
@@ -131,9 +128,9 @@
         v-if="(0 === currentList.length ) && false === isLoading"
         class="list__item u-pv-0_5 border--top color--grey">
         {{ Translator.trans('no.data') }}
-      </div>
+      </div><!--
```
Comment (salisdemos) [inline client/js/components/map/admin/AdminLayerList.vue:131]:
```suggestion
      </div>
```

File: client/js/components/map/admin/AdminLayerList.vue [hunk 2]
```diff
@@ -131,9 +128,9 @@
...
 
-      <h3 class="u-mt">
+   --><h3 class="u-mt">
```
Comment (salisdemos) [inline client/js/components/map/admin/AdminLayerList.vue:133]:
```suggestion
      <h3 class="u-mt">
```

**Review by elmasdemos — APPROVED** [review]
lgtm, code review

**Pull request merged**